installing/installing_cloud_other/verify-installation-cloud-other.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="verify-installation-cloud-other"]
+= Verifying installation of RHACS Cloud Service on other platforms
+include::modules/common-attributes.adoc[]
+:context: verify-installation-cloud-other
+
+toc::[]
+
+[role="_abstract"]
+Provides steps to verify that {product-title-managed-short} is properly installed.
+
+:FeatureName: {product-title-managed-short}
+include::snippets/field-trial.adoc[leveloffset=+1]
+
+To verify installation, access your ACS Console from the {cloud-console}. If the system is operating correctly, the Dashboard will display the number of clusters that {product-title-managed-short} is monitoring, along with information about nodes, deployments, images, and violations.
+
+If there is no data in the ACS Console, check the following:
+
+* You created an init bundle in the ACS Console, downloaded it, and used it on your secured cluster. ##Is this correct?##
+* ##How can they check that the Central API Endpoint is configured correctly on the secured cluster? Is there some way to see the address/port after configuration is complete?##
+* In the secured cluster, under *Networking* -> *Services*, verify that the following services are installed:
+** scanner
+** scanner-db
+** sensor
```

Comment:
What if these services are not installed?

Reply:
rewritten per Eng